sycl/doc/design/spirv-extensions/SPV_INTEL_function_pointers.asciidoc

```diff
@@ -39,14 +40,16 @@ If you are interested in using this feature in your software product, please let
 
 [width="40%",cols="25,25"]
 |==================================
-| Last Modified Date | {docdate}
-| Revision           | I
+| Last Modified Date | 2025-03-13
+| Revision           | 10
 |==================================
 
 == Dependencies
 
-This extension is written against the SPIR-V Specification, Version 1.4
-Revision 1, Unified.
+This extension is written against the SPIR-V Specification, Version 1.6
+Revision 5, Unified.
+
+This extension interacts with link:https://github.com/KhronosGroup/SPIRV-Registry/blob/main/extensions/KHR/SPV_KHR_untyped_pointers.asciidoc[*SPV_KHR_untyped_pointers*] extension.
```

Comment:
Nitpick: It'd be best to link to the rendered HTML specification rather than the asciidoc source.

```suggestion
This extension interacts with link:https://github.khronos.org/SPIRV-Registry/extensions/KHR/SPV_KHR_untyped_pointers.html[*SPV_KHR_untyped_pointers*] extension.
```